templates/log4j.properties.j2

```diff
@@ -14,10 +14,10 @@ zookeeper.tracelog.file=zookeeper_trace.log
 # Format is "<default threshold> (, <appender>)+
 
 # DEFAULT: console appender only
-log4j.rootLogger=${zookeeper.root.logger}
+#log4j.rootLogger=${zookeeper.root.logger}
 
 # Example with rolling log file
-#log4j.rootLogger=DEBUG, CONSOLE, ROLLINGFILE
+log4j.rootLogger=INFO, CONSOLE, ROLLINGFILE
```

Comment:
Why this is needed?

Reply:
To avoid extra logging with debug information.